### PR TITLE
docs: standalone panorama README for SWFW Hub

### PR DESCRIPTION
**Pull request by alperenkose**
## Description

Update standalone panorama example README for SWFW Hub

## Types of changes

- Documentation

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: examples/standalone_panorama/README.md [hunk 1]
```diff
@@ -21,16 +17,24 @@ This is a non zonal deployment. The deployed infrastructure consists of:
   * a Network Security Group to give access to Panorama's public interface
 * a Panorama appliance with a public IP assigned to the management interface
 
+![standalone-panorama](https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/assets/6500664/a2cc53c1-d57e-440a-b0d6-838e2217a20f)
```
Comment (FoSix) [inline examples/standalone_panorama/README.md:20]: The image is transparent. I think we should change it to one with a solid background as it's unreadable when using a dark mode in GH
image

File: examples/standalone_panorama/README.md [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
+# Standalone Panorama Deployment
 
->Panorama is a centralized management system that provides global visibility and control over multiple Palo Alto Networks next generation firewalls through an easy to use web-based interface. Panorama enables administrators to view aggregate or device-specific application, user, and content data and manage multiple Palo Alto Networks firewalls—all from a central location.
+Panorama is a centralized management system that provides global visibility and control over multiple Palo Alto Networks next generation firewalls through an easy to use web-based interface. Panorama enables administrators to view aggregate or device-specific application, user, and content data and manage multiple Palo Alto Networks firewalls—all from a central location.
 
-An example of a Terraform module that deploys a Panorama appliance in Azure.
+The Terraform code presented here will deploy Palo Alto Networks Panorama management platform in Azure in management only mode (without additional logging disks). For option on how to add additional logging disks - please refer to panorama [module documentation](../../modules/panorama/README.md#inputs)
```
Comment (FoSix) [inline examples/standalone_panorama/README.md:9]:
```suggestion
The Terraform code presented here will deploy Palo Alto Networks Panorama management platform in Azure in management only mode (without additional logging disks). For option on how to add additional logging disks - please refer to panorama [module documentation](../../modules/panorama/README.md#input_logging_disks)
```

**Review by FoSix — CHANGES_REQUESTED** [review]
generally 👍🏻

File: examples/standalone_panorama/README.md (hunk 1 above)
Comment (migara) [reply examples/standalone_panorama/README.md]: Fixed

**Review by migara — APPROVED** [review]
LGTM

**Pull request merged**